detectors/node/opentelemetry-resource-detector-aws/src/detectors/index.ts

```diff
@@ -17,5 +17,6 @@
 export * from './AwsEc2Detector';
 export * from './AwsBeanstalkDetector';
 export * from './AwsEcsDetector';
-export * from './AwsEksDetector';
+export { AwsEksDetector, awsEksDetector } from './AwsEksDetector';
```

Comment:
note for reviewer: https://github.com/open-telemetry/opentelemetry-js/issues/4186